packages/benchmark-test/package.json

```diff
@@ -1,9 +1,27 @@
 {
-    "name": "benchmark",
-    "version": "1.0.0",
-    "type": "module",
-    "scripts": {
-        "test": "yarn rollup -c && eshost -h v8,xs dist/bundle.js"
-    },
-    "license": "MIT"
+  "name": "@agoric/benchmark-test",
+  "version": "1.0.0",
+  "type": "module",
+  "scripts": {
+    "build": "exit 0",
+    "prepack": "tsc --build tsconfig.build.json",
+    "postpack": "git clean -f '*.d.ts*' '*.tsbuildinfo'",
+    "test": "yarn rollup -c && eshost -h 'V8,Moddable XS' dist/bundle.js",
```

Comment:
I didn't do any of the "Pre-reqs of running yarn test" instructions from #10975 , because I was curious how far I'd get with my existing configuration. I still have not tried that, so I don't know what difference it would make. That said:

By debugging into eshost code to figure out why it was failing, it was not finding the names "v8" or "xs". The names it seemed to want instead were "V8" and "Moddable XS". I tried it and it worked, but I suspect this is not a good change. I'm just reporting it as the outcome of my experiment.